index.js

```diff
@@ -62,9 +56,16 @@ GitHub.prototype = {
       cb = data;
       data = null;
     }
+
+    cb = typeof cb === 'function' ? cb : function noop () {};
```

Comment:
If the test coverage tool doesn't catch that there are 2 code branches in the `?:` operator, then that's an issue with the coverage tool. Also, you don't have to test to make sure that the `cb` becomes `noop`, just having a test that doesn't pass in a `cb` will work fine.